cabal-install/src/Distribution/Client/ProjectConfig.hs

```diff
@@ -1009,7 +1009,7 @@ writeProjectConfigFile file =
 readGlobalConfig :: Verbosity -> Flag FilePath -> Rebuild ProjectConfig
 readGlobalConfig verbosity configFileFlag = do
   config <- liftIO (loadConfig verbosity configFileFlag)
-  configFile <- liftIO (getConfigFilePath configFileFlag)
+  configFile <- liftIO (getConfigFilePath verbosity configFileFlag)
```

Comment:
The fact that it has to be modified in several places makes me thing that we'll get duplicate messages. Can you check this? Maybe the warning should be pulled up the call graph closer to `loadConfig` after all... (`extendToEffectiveConfig` or so)

Reply:
I looked into this when I implemented the original warning, and found it was not realistic. Cabal loads the config file in too many ad hoc places. Only solution would be something like a global IORef, but this does not work when cabal launches additional instances as subprocesses.

In practice, `cabal path` warns many times, but other commands at most twice.

Reply:
Fair enough